alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -124,12 +123,12 @@ impl OnResize for Pty {
 }
 
 fn cmdline(config: &PtyConfig) -> String {
-    let default_shell = Program::Just("powershell".to_owned());
+    let default_shell = Shell { program: "powershell".to_owned(), args: Vec::new() };
     let shell = config.shell.as_ref().unwrap_or(&default_shell);
 
-    once(shell.program())
-        .chain(shell.args().iter().map(|a| a.as_ref()))
-        .collect::<Vec<_>>()
+    once(shell.program.as_str())
+        .chain(shell.args.iter().map(|s| s.as_str()))
+        .collect::<Vec<&str>>()
```

Comment:
Is this actually required?

Reply:
We need to build a `String`, the only wait to join is from `slice`, and we need a `[&str]`, since it's the only thing implemented. The Windows API demands `String` to be passed.

Reply:
changed back to `Vec<_>`.